src/plugman/createpackagejson.js

```diff
@@ -39,18 +40,17 @@ function createPackageJson (plugin_path) {
         platforms: pluginInfo.getPlatformsArray()
     };
 
-    fs.writeFile(path.join(__dirname, 'defaults.json'), JSON.stringify(defaults), 'utf8', function (err) {
-        if (err) throw err;
-        events.emit('verbose', 'defaults.json created from plugin.xml');
-        var initFile = require.resolve('./init-defaults');
-        var dir = process.cwd();
+    return fs.writeFile(path.join(__dirname, 'defaults.json'), JSON.stringify(defaults), 'utf8')
```

Comment:
`fs.writeFile` needs to be wrapped to return a promise, doesn't it?

Reply:
It's the one from `fs-extra`. That's already promisified.